Xamarin.ExposureNotification/ExposureNotification.android.cs

```diff
@@ -45,6 +46,45 @@ static Task PlatformStop()
 		static async Task<bool> PlatformIsEnabled()
 			=> await Instance.IsEnabledAsync();
 
+		public static void ConfigureBackgroundWorkRequest(Action<PeriodicWorkRequest.Builder> requestBuilder, TimeSpan repeatInterval)
+		{
+			if (requestBuilder == null)
+				throw new ArgumentNullException(nameof(requestBuilder));
+			if (repeatInterval == null)
+				throw new ArgumentNullException(nameof(repeatInterval));
+
+			bgRequestBuilder = requestBuilder;
+			bgRepeatInterval = repeatInterval;
+		}
+
+		static Action<PeriodicWorkRequest.Builder> bgRequestBuilder = b =>
+			b.SetConstraints(new Constraints.Builder()
+				.SetRequiresBatteryNotLow(true)
+				.SetRequiresDeviceIdle(true)
+				.SetRequiredNetworkType(NetworkType.Connected)
+				.Build());
+
+		static TimeSpan bgRepeatInterval = TimeSpan.FromHours(6);
+
+		static Task PlatformScheduleFetch()
+		{
+			var workManager = WorkManager.GetInstance(Essentials.Platform.AppContext);
+
+			var workRequestBuilder = new PeriodicWorkRequest.Builder(
+				typeof(BackgroundFetchWorker),
+				bgRepeatInterval);
+
+			bgRequestBuilder.Invoke(workRequestBuilder);
+			
+			var workRequest = workRequestBuilder.Build();
+
+			workManager.EnqueueUniquePeriodicWork("exposurenotification",
+				ExistingPeriodicWorkPolicy.Replace,
```

Comment:
I think that's ok because 1) in practice most users aren't going to do this, and 2) if they do, it's still a fairly aggressive schedule - in Google's sample they only run once per day.